docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
 
-The Gravitino server log configuration file is `conf/log4j2.properties`. Gravitino uses Log4j2 as the Logging system. You can [Log4j2](https://logging.apache.org/log4j/2.x/) to do the log configuration.
+### Configure the Gravitino server
 
-#### Configure the Apache Gravitino server environment
+The `conf/gravitino.conf` file provides basic configurations for the Gravitino server.
+To configure the Gravitino server, you can update this file.
+For a full list of the Gravitino server configurations, see [Gravitino Server Configurations](./gravitino-server-config.md).
 
-The Gravitino server environment configuration file is `conf/gravitino-env.sh`. Gravitino exposes several environment variables. You can modify them in this file.
+### Configure the Gravitino server log
 
-#### Configure Apache Gravitino catalogs
+Gravitino uses [Log4j2](https://logging.apache.org/log4j/2.x/) as the logging system.
+The `conf/log4j2.properties` file provides the Gravitino server log configurations.
+To configure the Gravitino server log, you can update this file.
 
-Gravitino supports multiple catalogs. You can configure the catalog-level configurations by modifying the related configuration file in the `catalogs/<catalog-provider>/conf` directory. The configurations you set here apply to all the catalogs of the same type you create.
+### Configure the Gravitino server environment variables
 
-For example, if you want to configure the Hive catalog, you can modify the file `catalogs/hive/conf/hive.conf`. The detailed configurations are listed in the specific catalog documentation.
+Gravitino exposes several environment variables through the `conf/gravitino-env.sh` file.
+To configure these environment variables, you can update this file.
+
+### Configure the Gravitino catalogs
+
+Gravitino supports multiple catalogs.
+You can configure the catalog-level configurations by updating the related configuration file in the `catalogs/<catalog-provider>/conf` directory.
+For example, the `catalogs/hive/conf/hive.conf` file provides configurations for the Hive catalog.
+The configurations you set in the catalog configuration file will apply to all the catalogs of the same type that you created.
+For detailed configurations about each catalog, see related catalog documentation.
 
 :::note
-Gravitino takes the catalog configurations in the following order:
+Gravitino takes the catalog configurations in the following order of precedence:
 
-1. Catalog `properties` specified in catalog creation API or REST API.
+1. Catalog `properties` specified in the catalog creation API or REST API.
 2. Catalog configurations specified in the catalog configuration file.
 
-The catalog `properties` can override the catalog configurations specified in the configurationfile.
+The catalog `properties` overrides the catalog configurations specified in the configuration file.
```

Comment:
2. Catalog configuration file settings defined in the catalog configuration file. 
Catalog `properties` override the configuration defined in the catalog configuration file.

Reply:
1. why the second uses catalog configuration file settings ... instead of catalog configurations

Reply:
The word configurations is misused. The noun configuration can be countable or uncountable, i.e. the plural form of configuration is configuration. You might use configurations to refer to multiple configuration files, but even that is quite rare.